### PR TITLE
Fix typo in doc

### PR DESCRIPTION
**Pull request by ono-max**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs-dev/user-guide/cancelling-a-deployment.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: >
 
 A running deployment can be cancelled from web UI at the deployment details page.
 
-If the application rollback is enabled in the deployment configuration, the rollback process will be executed after the cancelling. You can also explicitly specify to rollback after the cancelling or not from the web UI by clicking on `▼` mark of the `CANCEL` button to select your option.
+If the application rollback is enabled in the deployment configuration, the rollback process will be executed after the cancelling. You can also explicitly specify to rollback after the cancelling or not from the web UI by clicking on `▼` mark of the `CANCEL with Rollback` button to select your option.
```
Comment (nakabonne) [inline docs/content/en/docs-dev/user-guide/cancelling-a-deployment.md:11]: Good catch. Possibly what this sentence tries to say is:

```suggestion
If the application rollback is enabled in the deployment configuration, the rollback process will be executed after the cancelling. You can also explicitly specify to rollback after the cancelling or not from the web UI by clicking on `▼` mark on the right side of the `CANCEL with Rollback` button to select your option.
```

Comment (ono-max) [reply docs/content/en/docs-dev/user-guide/cancelling-a-deployment.md:11]: Thanks!

**Comment by ono-max** [comment]
I'll update others, too.

File: docs/content/en/docs-dev/user-guide/cancelling-a-deployment.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ description: >
 
 A running deployment can be cancelled from web UI at the deployment details page.
 
-If the application rollback is enabled in the deployment configuration, the rollback process will be executed after the cancelling. You can also explicitly specify to rollback after the cancelling or not from the web UI by clicking on `▼` mark of the `CANCEL` button to select your option.
+If the application rollback is enabled in the deployment configuration, the rollback process will be executed after the cancelling. You can also explicitly specify to rollback after the cancelling or not from the web UI by clicking on `▼` mark on the right side of the `CANCEL with Rollback` button to select your option.
```
Comment (nakabonne) [inline docs/content/en/docs-dev/user-guide/cancelling-a-deployment.md:11]: Sorry I made a mistake
```suggestion
If the application rollback is enabled in the deployment configuration, the rollback process will be executed after the cancelling. You can also explicitly specify to rollback after the cancelling or not from the web UI by clicking on `▼` mark on the right side of the `CANCEL` button to select your option.
```

**Comment by nakabonne** [comment]
Nice
/approve

**Pull request merged**